libs/gltfio/src/AssetLoader.cpp

```diff
@@ -357,6 +359,12 @@ struct FAssetLoader : public AssetLoader {
 
 FILAMENT_DOWNCAST(AssetLoader)
 
+FAssetLoader::~FAssetLoader() {
+    if (mDummyBufferObject) {
+        mEngine.destroy(mDummyBufferObject);
+    }
+}
```

Comment:
was this just leaking before?  Also is it safe to do this in the FAssetLoader dtor -- i.e. is it guaranteed to be called on the correct thread?

Reply:
actually, I misread the code.  It was properly cleaned up before.  I removed all the dummy buffer changes.